shared-doc/build-and-run-the-quickstart-with-bootable-jar.adoc

```diff
@@ -25,6 +25,11 @@ ifdef::reactive-messaging[]
                               <layer>cloud-server</layer>
                               <layer>h2-default-datasource</layer>
                               <layer>microprofile-reactive-messaging-kafka</layer>
+endif::[]
+ifdef::microprofile-lra[]
+                              <layer>jaxrs-server</layer>
```

Comment:
Personally I would prefer to have here an abstract `<layers>...<layers/>`, which suits all and needs no maintenance. @zhantemirov @xstefank WDYT?

Reply:
If it doesn't increase the server build time significantly, I'm OK with that change

Reply:
I was actually referring to the docs only, replace the layers config in the XML snippet with what I proposed above. This would be done on all asciidocs having the plugins config showing up layers XML, for both wildfly and wildfly-jar.

Reply:
There is no easy way to override this. Unfortunately for LRA and RM we need to have different layers.

Reply:
https://issues.redhat.com/browse/WFLY-18819